src/pmdas/uwsgi/pmdauwsgi.1

```diff
@@ -0,0 +1,80 @@
+'\"macro stdmacro
+.\"
+.\" Copyright (c) 2024 Red Hat.
+.\" Copyright (c) 2024 Nikhil Jain <nikjain@redhat.com>
+.\"
+.\" This program is free software; you can redistribute it and/or modify it
+.\" under the terms of the GNU General Public License as published by the
+.\" Free Software Foundation; either version 2 of the License, or (at your
+.\" option) any later version.
+.\"
+.\" This program is distributed in the hope that it will be useful, but
+.\" WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+.\" or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+.\" for more details.
+.\"
+.\"
+.TH PMDAUWSGI 1 "PCP" "Performance Co-Pilot"
+.SH NAME
+\f3pmdauwsgi\f1 \- uwsgi performance metrics domain agent (PMDA)
+.SH DESCRIPTION
+\f3pmdauwsgi\f1 is a Performance Metrics Domain Agent (\s-1PMDA\s0) which
+exports performance metrics from uwsgi.
+uWSGI is often used in conjunction with web servers such Nginx, which 
+offer direct support for uWSGI's native uwsgi protocol, to serve 
+Python web applications such as Django.
+.PP
+\fBpmdauwsgi\fP reads an optional ini-style configuration file:
+.IP
+.PD 0
+.IP
+.I \f(CR$PCP_PMDAS_DIR\fP/uwsgi/uwsgi.conf
```

Comment:
I recommend moving this script to a new home below /etc/pcp/uwsgi ($PCP_SYSCONF_DIR/uwsgi) - reason being, rpm-ostree based systems do not always have persistent /var contents so custom configurations here may be lost.
Other agents you probably came across often now use symlinks below /var into /etc to tackle this issue in a backward compatible way - but you can start out on the right foot since its a new agent.

Reply:
Sure. Thanks for pointing this out.